backend/MASZ/Exceptions/EmbedCreator.cs

```diff
@@ -328,13 +328,13 @@ public static async Task<EmbedBuilder> CreateInternalAutomodEmbed(this AutoModer
                 false
             );
 
-            if (! string.IsNullOrWhiteSpace(autoModerationEvent.MessageContent)) {
-                embed.AddField(
-                    translator.T().MessageContent(),
-                    autoModerationEvent.MessageContent,
-                    false
-                );
-            }
+            //if (! string.IsNullOrWhiteSpace(autoModerationEvent.MessageContent)) {
+            //    embed.AddField(
+            //        translator.T().MessageContent(),
+            //        autoModerationEvent.MessageContent,
+            //        false
+            //    );
+            //}
```

Comment:
I think this can stay in here as it tells moderators what the automoderator punished without having to login to the website.

Reply:
Alright, will revert the change